build_package.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+VERSION=$(<VERSION)
+echo "Using version $VERSION"
+￼
+PBR_VERSION=$VERSION python3 setup.py sdist bdist_wheel
+￼
+if [ -f "dist/faucet-$VERSION.tar.gz" ]; then
```

Comment:
leave success as the end of the script, so exit out here on failure cases -- there's multiple reasons to fail, but only one path to success.

Reply:
Done